sdk/ai/azure-ai-agents/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 # Release History
 
+## 1.2.0b5 (Unreleased)
+
+### Features Added
+
+### Bugs Fixed
+- Added `RunStepDeltaChunk` to `StreamEventData` model
```

Comment:
Please mention the GitHub issue in parenthesis, similar to what we've done before.